scout/server/blueprints/institutes/controllers.py

```diff
@@ -269,6 +269,9 @@ def populate_institute_form(form, institute_obj):
     form.institutes.choices = institutes_tuples
     form.coverage_cutoff.default = institute_obj.get("coverage_cutoff")
     form.frequency_cutoff.default = institute_obj.get("frequency_cutoff")
+    form.show_all_cases_categories.data = institute_obj.get("show_all_cases_categories") or [
```

Comment:
I know you mean it as a general term, but since we have variant categories, also indirectly displayed via the links, best be explicit about case status. 
```suggestion
    form.show_all_cases_status.data = institute_obj.get("show_all_cases_status") or [
```

Reply:
Agreed!